modules/data.remote/inst/gee2pecan_l8.py

```diff
@@ -0,0 +1,191 @@
+"""
+Extracts Landsat 8 surface reflactance band data from Google Earth Engine and saves it in a netCDF file
+
+Requires Python3
+
+Bands retrieved: B1, B2, B3, B4, B5, B6, B7, B10, B11 along with computed NDVI
+
+If ROI is a Point, this function can be used for getting SR data from Landsat 7, 5 and 4 as well.
+
+Author: Ayush Prasad
+"""
+from gee_utils import create_geo, get_siteaoi, get_sitename
+import ee
+import pandas as pd
+import geopandas as gpd
+import datetime
+import os
+import xarray as xr
+import numpy as np
+import re
+
+ee.Initialize()
+
+
+def gee2pecan_l8(geofile, outdir, start, end, qc=1):
+    """
+    Extracts Landsat 8 SR band data from GEE
+
+    Parameters
+    ----------
+    geofile (str) -- path to the file containing the name and coordinates of ROI, currently tested with geojson. 
+    
+    outdir (str) -- path to the directory where the output file is stored. If specified directory does not exists, it is created.
+  
+    start (str) -- starting date of the data request in the form YYYY-MM-DD
+    
+    end (str) -- ending date areaof the data request in the form YYYY-MM-DD
+
+    ic (str) -- image collection id of the Landsat SR product from Google Earth Engine
+
+    vi (bool) -- set to True if NDVI needs to be calculated
+
+    qc (bool) -- uses the cloud masking function if set to True
+
+    bands (list of str) -- bands to be retrieved. Default: B5, B4
+
+    Returns
+    -------
+    Nothing:
+            output netCDF is saved in the specified directory. 
+ 
+    """
+
+    # scale (int) Default: 30
+    scale = 30
```

Comment:
could you explain what this is and if it could be a function arg if we want the flexibility for users to change

Reply:
It's the pixel resolution of the image https://developers.google.com/earth-engine/scale . I have sort of used a generic value here and in the other functions. This will probably need to be configured depending upon the use case. I didn't pass it as an argument because `gee2pecan_s2` does not do it currently (it's listed in the TODO). I can make it configurable there as well? Overall, I think all the gee extraction functions should have the same type of arguments.

Reply:
>I think all the gee extraction functions should have the same type of arguments.
Agreed. Then, they could also be explained in the book-documentation.

It would be great if you could make it configurable, even though it is not currently used in gee2pecan_s2.

Also, could you add it to the book documentation with the link you shared here? it was very informative, thank you.

Reply:
So I have 2 options here,
1. I only make it configurable in l8. In this case, I will have to add an additional `if` in `get_remote_data` which would check if the scale is passed in the arguments and then call the appropriate function.
2. I can also make it configurable in s2 even though it's not required (currently). This will not require adding the above `if`. 
BTW AppEEARS doesn't have this option to specify the scale. There we need to specify the type of projection required

Reply:
2nd option sounds good to me as well